lib/datasource/index.ts

```diff
@@ -28,6 +29,19 @@ function load(datasource: string): Promise<Datasource> {
 
 type GetReleasesInternalConfig = GetReleasesConfig & GetPkgReleasesConfig;
 
+function applyReplacements(
+  dep: ReleaseResult,
+  config: GetReleasesInternalConfig
+): ReleaseResult {
+  if (config.replacementName && config.replacementVersion) {
+    const ret = clone(dep);
```

Comment:
Why we need to clone here? 🤔

Reply:
If I recall correctly, it was to avoid triggering [no-param-reassign](https://eslint.org/docs/rules/no-param-reassign) in our eslint config.

Reply:
so we don't need a deep copy and use:

```js
  return { ...dep, replacementName: config.replacementName, replacementVersion: config.replacementVersion };
```